src/tools/gophish_export.py

```diff
@@ -253,6 +253,86 @@ def get_application(rawEvent):
     return application
 
 
+def find_unique_target_clicks_count(clicks):
+    """Return the number of unique clicks in a click dict."""
+    uniq_list = list()
+    for click in clicks:
+        if click["user"] not in uniq_list:
+            uniq_list.append(click["user"])
+    return len(uniq_list)
+
+
+def write_campaign_summary(api, assessment_id):
+    """Output a campaign summary report to JSON, console, and a text file."""
+    campaign_ids = get_campaign_ids(api, assessment_id)
+    campaign_data_template = "campaign_data.json"
+    campaign_summary_json = assessment_id + "_campaign_data.json"
+    campaign_summary_textfile = (
+        assessment_id + "_summary_" + str(datetime.now()) + ".txt"
+    )
+
+    with open(campaign_data_template) as template:
+        campaign_data = json.load(template)
+
+    logging.info("Writing campaign summary report to %s" % campaign_summary_textfile)
```

Comment:
Use preferred formatting syntax in logging functions:
```suggestion
    logging.info("Writing campaign summary report to %s", campaign_summary_textfile)
```

Reply:
See 5816a24f37e9a600f2b55946de18b7acb8d0da07